src/core/acroform/PDFAcroField.ts

```diff
@@ -96,6 +110,24 @@ class PDFAcroField {
     return DA?.asString();
   }
 
+  setFontSize(fontSize: number) {
```

Comment:
Please add some unit tests for this method to https://github.com/Hopding/pdf-lib/blob/master/tests/core/acroform/PDFAcroField.spec.ts. Parsing logic like this is error prone so it's important to test carefully to avoid regressions and missed edge cases.